packages/capabilities/src/types.ts

```diff
@@ -458,9 +492,8 @@ export type ServiceBlob = InferInvokedCapability<typeof W3sBlobCaps.blob>
 export type BlobAllocate = InferInvokedCapability<typeof W3sBlobCaps.allocate>
 export type BlobAccept = InferInvokedCapability<typeof W3sBlobCaps.accept>
 
-export type BlobMultihash = Uint8Array
```

Comment:
This is a breaking change, I think this is imported somewhere else IIRC. Either should be kept, or need to be flagged

Reply:
perhaps is just in `upload-api`, but honestly I would keep this or signal a Breaking Change